lib/PuppeteerSharp/Page.cs

```diff
@@ -2529,9 +2538,8 @@ public void Dispose()
         /// the garbage collector can reclaim the memory that the <see cref="Page"/> was occupying.</remarks>
         /// <param name="disposing">Indicates whether disposal was initiated by <see cref="Dispose()"/> operation.</param>
         protected virtual void Dispose(bool disposing) => _ = DisposeAsync();
-        #endregion
 
```

Comment:
I recommend removing a newline to satisfy the Style Cop rule for newlines.

Reply:
done!